sql/mysql_db_defaultdata.sql

```diff
@@ -59,7 +59,8 @@ INSERT INTO `configuration` (`name`, `value`, `type`, `public`, `category`, `des
 ('openid_clientid', '""', 'string', '0','Misc', 'OpenID Connect client id'),
 ('openid_clientsecret', '""', 'string', '0', 'Misc', 'OpenID Connect client secret'),
 ('data_source', '0', 'int', '0', 'Misc', 'Source of data. Choices: 0 = all local, 1 = configuration data external, 2 = configuration and live data external'),
-('ip_autologin', '0', 'bool', '0', 'Misc', 'Enable to skip the login page when using IP authentication.');
+('ip_autologin', '0', 'bool', '0', 'Misc', 'Enable to skip the login page when using IP authentication.'),
+('team_column_width', '0', 'int', '0', 'Misc', 'Maximum width of team column on scoreboard. Leave 0 for no maximum.');
```

Comment:
How about putting it in display options?

Reply:
Changed in 2a5408eb58df7c7126c411d447e4a3578d0cc0d8